app/controllers/trainees/course_details_controller.rb

```diff
@@ -26,6 +26,7 @@ def update
       )
 
       if @course_details_form.stash_or_save!
+        Trainees::SetAcademicCycles.call(trainee: trainee).save!
```

Comment:
Is the controller the appropriate place for this to happen?

Reply:
I was umming and ahhing about that myself! I noticed that if we create a trainee 'by hand' it doesnt get a start_academic_cycle. Since we use the course details to calculate the start_academic_cycle I thought here was a good place to put it, should it go in the form instead?